README.md

```diff
@@ -166,28 +151,24 @@ This is not a normal shell, so try not to type Ctrl+C.
 /srv/erlang[1]>
 ```
 
-There are a few caveats to using this shell right now, so you'll have to be
-careful when you use it:
+There are a few caveats to using this shell for now:
 
-1. `Ctrl+C Ctrl+C` exits the Erlang VM and will reboot or hang your system
-   depending on how `erlinit` is configured.
-2. Because of the `Ctrl+C` caveat, you can't easily break out of long running
-   programs. As a workaround, start another shell using `Ctrl+G` and `kill` the
-   offending program.
-3. Commands are run asynchronously. This is unexpected if you're used to a
-   regular shell. For most commands, it's harmless. One side effect is that if a
-   command changes the current directory, it could be that the prompt shows the
-   wrong path.
+1. `Ctrl+C Ctrl+C` exits the Erlang VM and will reboot or hang your system depending on how `erlinit` is configured.
```

Comment:
I recently changed this by adding `+Bc` to the `vm.args` in the new project generator. A lot of people probably don't have this, but it may be worth mentioning.